src/status_im/native_module/impl/module.cljs

```diff
@@ -156,7 +156,7 @@
   "Removes duplicates by [jail path] keys, remains the last one."
   [[all-keys calls] {:keys [jail-id path] :as call}]
   (if (and (contains? all-keys [jail-id path])
-           (not= (second path) :subscription))
+           (not (#{:subscription :preview} (last path))))
```

Comment:
I actually think it's safer to do it the other way, eq only deduplicate based on `[jail-id path]` when we are sure we want to do it (eq suggestions) then the other way around.
But not a big deal for now